### PR TITLE
CMDCT-3202: Target Populations Alert Text (SAR)

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

If a user has not selected any target populations on their MFP Work Plan, the following alert text should display under the `Recruitment, Enrollment, and Transitions` pages of the SAR:

Screenshot 2024-02-22 at 11 34 34 AM

### Related ticket(s)

CMDCT-3202

---
### How to test

- Log in as a state user
- Create a WP (selecting `No` on all of the target populations)
- Submit that WP
- Log in as an admin
- Approve the WP
- Log back in as the state user
- Create a SAR

You should see the alert text under all of the `sar/recruitment-enrollment-transitions/` pages.

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/forms/Form.tsx [hunk 1]
```diff
@@ -56,6 +56,11 @@ export const Form = ({
 
   const reportWithSubmittedStatus = reportStatus === ReportStatus.SUBMITTED;
 
+  // For the SAR RE&T sections, display an alert if the MFP WP does not contain any target populations
+  const displayRetError =
+    report?.populations?.length === 0 &&
+    location.pathname.startsWith("/sar/recruitment-enrollment-transitions");
```
Comment (gmrabian) [inline services/ui-src/src/components/forms/Form.tsx:62]: I'm getting `false` for this. Specifically the length of populations comes back as `4`. Seems like it's still counting them even if they are selected as "no". What are others experiencing? 


I used the postman collection to create the WP and SAR.

Comment (karla-vm) [reply services/ui-src/src/components/forms/Form.tsx:62]: I'm taking a look at that now! 👀

Comment (karla-vm) [reply services/ui-src/src/components/forms/Form.tsx:62]: @gmrabian can you try this again? I'm not able to replicate your issue, maybe the Postman collection was adding target populations?

**Review by ailZhou — APPROVED** [review]
tested on deploy site. manually entered data and the alert text showed up

File: services/ui-src/src/components/forms/Form.tsx (hunk 1 above)
Comment (gmrabian) [reply services/ui-src/src/components/forms/Form.tsx:62]: Bug went away 🤷

**Review by gmrabian — APPROVED** [review]
working on local
(this text `Your associated MFP Work Plan does not contain any target populations.` is black in the PDF, in case we want that to be red as well)

**Pull request merged**